lib/capistrano/tasks/nvm.rake

```diff
@@ -9,17 +9,21 @@ namespace :nvm do
 
       nvm_node_path = fetch(:nvm_node_path)
       nvm_node_path = [nvm_node_path] unless nvm_node_path.is_a?(Array)
+      nvm_verb = fetch(:nvm_verb).to_s
 
       unless test(nvm_node_path.map {|p| "[ -d #{p} ]" }.join(" || "))
-        error "nvm: #{nvm_node} is not installed or not found in any of #{nvm_node_path.join(" ")}"
-        exit 1
+        if nvm_verb == 'use'
+          error "nvm: #{nvm_node} is not installed or not found in any of #{nvm_node_path.join(" ")}"
+          exit 1
+        end
       end
     end
   end
 
   task :map_bins do
     SSHKit.config.default_env.merge!({ node_version: "#{fetch(:nvm_node)}" })
     nvm_prefix = fetch(:nvm_prefix, -> { "#{fetch(:tmp_dir)}/#{fetch(:application)}/nvm-exec.sh" } )
+    SSHKit.config.default_env[:nvm_verb] = fetch(:nvm_verb).to_s
```

Comment:
Now that I look at this again, there's no need to add an environment variable for `nvm_verb`. Below you can simply use `fetch(:nvm_verb)` instead of `$NVM_VERB`.